README.md

```diff
@@ -200,25 +200,25 @@ Note that any communication with the probe is synchronous.
 
 ## Screenshots
 
-Screenshots are saved under the path taken from `IDEPROBE_SCREENSHOTS_DIR` environment variable for which the default value is `/tmp/ide-probe/screenshots`. Inside this directory
+The default folder for saving photos is `/tmp/ide-probe/screenshots`. Inside this directory
 screenshots are organised into `test-suite/test-case` subfolders if these values are detectable by the plugin.
-Otherwise, the photos are saved directly into `IDEPROBE_SCREENSHOTS_DIR`. 
-The only way to override `IDEPROBE_SCREENSHOTS_DIR` is by configuring `probe.screenshot.path` in the fixture, either through the `.conf` file or in code.
+Otherwise, the photos are saved directly into `/tmp/ide-probe/screenshots`. 
+The only way to override this default directory is by configuring `probe.screenshot.path` in the fixture, either through the `.conf` file or in code.
 Overriding with `.conf` file looks like: 
 ```
 probe {
    // ...
    paths.screenshots = ${?MY_IDEPROBE_SCREENSHOTS_DIR}
 }`
 ```
-Thanks to HOCON feature, it is possible to populate any config value from environment variable and additionally make it optional
+Thanks to HOCON feature, it is possible to populate any config value from environment variable. Additionally, it is achievable to make this value optional
 with question mark used before the name of the environment variable - if `MY_IDEPROBE_SCREENSHOTS_DIR`  does not exist at runtime
 `probe.screenshot.path` will use default value without any error.
 For non `.conf` scenario, user would call: 
 ```
 fixture.withPaths(IdeProbePaths(/*construct the instance passing, among others, a screenshots path that is most suitable for you*/))
 ```
-
+Screenshots feature is only available with Xvfb display mode. 
```

Comment:
Plus that screenshots are taken on every AWT event + on exit + when explicitly requested via `probe.screenshot()`